jobserver/templates/workspace_detail.html

```diff
@@ -27,7 +27,7 @@
       <h3>{{ workspace.name }}</h3>
 
       <div class="d-flex">
-        <a class="btn btn-primary mr-2" href="{% url 'workspace-logs' name=workspace.name %}">
+        <a class="btn btn-primary mr-2" href="{% url 'workspace-logs' org_slug=workspace.project.org.slug project_slug=workspace.project.slug workspace_slug=workspace.name %}">
```

Comment:
Might benefit from a helper method to generate the URL?

Reply:
Definitely, there's a few little bits like this which I need to go and clean up in subsequent PRs.